Source/Extensions/Blazorise.Markdown/wwwroot/markdown.js

```diff
@@ -1,3 +1,8 @@
+import "./vendors/easymde.js?v=1.1.0.0";
+import "./vendors/highlight.js?v=1.1.0.0";
```

Comment:
This both has the advantage that we test and make blazorise work with a certain mde version, but also the disavantage that users can't easily upgrade their markdown js to a new version, right?

Reply:
Well... some fix in the library the user really needed, :P Anyway as long as we give continuos quick support It's no biggy, we update and launch a patch with the new version I guess